manala.graylog_sidecar/defaults/main.yml

```diff
@@ -0,0 +1,14 @@
+---
+
+# Install
+manala_graylog_sidecar_install_version: ~
+manala_graylog_sidecar_install_packages:
+  - https://github.com/Graylog2/collector-sidecar/releases/download/{{ manala_graylog_sidecar_install_version }}/graylog-sidecar_{{ manala_graylog_sidecar_install_version }}-1_amd64.deb
```

Comment:
Should we have an `manala_graylog_sidecar_install_default_package` instead ? I am afraid that this URL can change in near future